doc/api/fs.md

```diff
@@ -4209,7 +4209,7 @@ this API: [`fs.writev()`][].
 
 The `fs.promises` API provides an alternative set of asynchronous file system
 methods that return `Promise` objects rather than using callbacks. The
-API is accessible via `require('fs').promises`.
+API is accessible via `require('fs').promises` or `require('fs/promises')`.
```

Comment:
Just leaving it at this makes it a bit confusing which should be used. Not sure I have a better suggestion. Is the intent to eventually deprecate `require('fs').promises`?

Reply:
I personally have no intention to deprecate it